src/views/Home/components/Hero/Hero.scss

```diff
@@ -25,8 +48,107 @@
     }
   }
 
+  // background
+  &__grid {
+    position: absolute;
+    transform: scaleY(-1);
+    z-index: -2;
+    bottom: 0;
+    left: 6vw;
+    max-width: calc(90% - 40vw);
+    max-height: 50%;
+    min-height: calc(20em + 60vh);
+
+    @include responsiveness.respond(tab-port) {
+      left: 0;
+      max-width: calc(20vw + 15vh);
+      min-height: 90%;
+      max-height: 100%;
+    }
+
+    @include responsiveness.respond(smallest) {
+      visibility: hidden;
+    }
+  }
+
+  &__computer {
+    position: absolute;
+    z-index: -1;
+    top: calc(-30em - 4vh);
+    width: 100%;
```

Comment:
What's the purpose of this?

Reply:
@tim-nguyen-cs not quite sure which line you're referring to here

Reply:
Oh, my bad! I was referring to the `max-width: 100%` and the `width: 100%`.

Reply:
ah that was a bit of a hacky workaround with responsiveness, I've since updated the `max-width` line with a different unit.